packages/opencensus-web-initial-load/src/index.ts

```diff
@@ -14,13 +14,6 @@
  * limitations under the License.
  */
 
-import { SpanContext } from '@opencensus/web-types';
-
-const IS_SAMPLED_BIT = 0x1;
-
-/** Returns whether sampling hint bit of the span context `options` is set. */
-export function isSampled(spanContext: SpanContext) {
-  const options = spanContext.options;
-  if (!options) return false;
-  return !!(options & IS_SAMPLED_BIT);
-}
+export { exportRootSpanAfterLoadEvent } from './export-initial-load';
```

Comment:
Same here on moving these to `index.ts` if we are just re-exporting them to keep compatibility

Reply:
I'm not sure to understand this. These exports are already in the `index.ts`